.vib/codeigniter/goss/vars.yaml

```diff
@@ -0,0 +1,22 @@
+binaries:
+  - composer
+  - php-fpm
+  - gosu
+  - mmdblookup
```

Comment:
Why adding mmdblookup here? Is it required for CodeIgniter somehow?

Reply:
Ok I removed it